docs/main_text.md

```diff
@@ -437,7 +437,15 @@ We then quantile normalize all samples as described above.
 
 We've made our analyses underlying processing choices and exploring test compendia available at our <a href = "https://github.com/AlexsLemonade/compendium-processing" target = "blank">`compendium-processing`</a> repository.
 
-### Download Folder
+### Collapsing by genus
+
+Microarray platforms are generally designed to assay samples from a specific species.
+In some cases, samples that are not from the species for which a microarray platform is specifically designed for are in publicly available data surveyed by refine.bio (e.g., samples labeled _Bos indicus_ will be run on _Bos taurus_ microarrays or mouse crosses that are not labeled _Mus musculus_ will be run on _Mus musculus_ microarrays). 
+When we encounter this in refine.bio, we will include samples from the species that differs from the platform species when the two species share a genus in the larger species' normalized compendium (e.g., _Bos indicus_ samples run on _Bos taurus_ microarrays are included in the _Bos taurus_ normalized compendium).
+This is generally a low number of samples compared to the total number of samples included in a normalized compendium.
+If you would like to filter a normalized compendium based on a sample's species label, you can utilize the `refinebio_organism` column in the metadata TSV file or the `.samples[].refinebio_organism` field in the metadata JSON file included as part of the download.
```

Comment:
```suggestion
If you would like to filter a normalized compendium based on a sample's species label, you can use the `refinebio_organism` column in the metadata TSV file or the `.samples[].refinebio_organism` field in the metadata JSON file included as part of the download.
```

I hate utilize most of the time. It might be okay here, so feel free to ignore my persnickitiness.